internals/notifier.py

```diff
@@ -41,26 +41,36 @@
     AppUser, BlinkComponent, FeatureOwner, UserPref)
 
 
+def _determine_milestone_string(ship_stages: list[Stage]) -> str:
+  """Determine the shipping milestone string to display in the template."""
+  has_desktop_mstone = False
```

Comment:
Much clearer to me, and not only that, but my version has the corner case of choosing an earlier android milestone (because it is the min), but not marking it with the `(android)` suffix if there is also a later desktop milestone that exists. So not just better, but more correct. 🙂